cg/meta/orders/schema.py

```diff
@@ -152,21 +152,21 @@ def validate(self, value):
 
     # required
     'name': validators.RegexValidator(NAME_PATTERN),
-    'capture_kit': validators.Any(CAPTUREKIT_OPTIONS),
+    'capture_kit': validators.Optional(TypeValidator(str, allow_none=True), None),
     'application': str,
     'sex': validators.Any(SEX_OPTIONS),
     'family_name': validators.RegexValidator(NAME_PATTERN),
     'priority': validators.Any(PRIORITY_OPTIONS),
-    'source': str,
+    'source': validators.Optional(TypeValidator(str, allow_none=True), None),
 
     # Required if data analysis in Scout
     'panels': ListValidator(str, min_items=0),
     # todo: find out if "Additional Gene List" is "lost in translation", implement in OP or remove from OF
     'status': validators.Any(STATUS_OPTIONS),
 
     # Required if samples are part of trio/family
-    'mother': validators.Optional(validators.RegexValidator(NAME_PATTERN), None),
-    'father': validators.Optional(validators.RegexValidator(NAME_PATTERN), None),
+    'mother': validators.Optional(TypeValidator(str, allow_none=True), None),
```

Comment:
So, this used to be a regexvalidator with a pattern. Now it is an optional string. Does that mean we cannot have an optional regex?

Reply:
Implemented now :)